t/exotic_names.t

```diff
@@ -45,7 +45,7 @@ sub caller3_ok {
         ),
     );
 
-    $expected =~ s/'/::/g if $] < 5.041_003;
+    $expected =~ s/'/::/g if $] < 5.037009 || $] >= 5.041_004;
```

Comment:
If the code will work for all versions, then we should remove the conditional here, so as to test on all versions, rather than add more gating?

Reply:
I wouldn't worry if new releases break against previous development cycles in the middle, for the sake of keeping conditions simpler. We may already have other things that break mid-cycle developments anyway.